mars/services/task/execution/ray/executor.py

```diff
@@ -258,6 +276,7 @@ async def execute_subtask_graph(
             elif output_count == 1:
                 output_object_refs = [output_object_refs]
             self._cur_stage_output_object_refs.extend(output_object_refs)
+            self._output_object_refs_nums.append(len(output_object_refs))
```

Comment:
Why not just iterate the `self._cur_stage_output_object_refs` to cancel the object refs?

Reply:
We can just cancel one object ref to cancel a ray task even if the it has multiple return object refs.

Reply:
Good to know this. Thanks.

Reply:
Can you add some comments for the `_output_object_refs_nums`?

Reply:
Ok.